### PR TITLE
General - add German loadout documentation

### PR DESCRIPTION
**Pull request by PervonHarke**
**When merged this pull request will:**
- rename multiple loadouts to add missing version identifiers to file names
- removed one unneeded/unfinished role from `deu_ca2010_fallschirmjaeger_flecktarn.sqf`
- added documentation for all German loadouts

### IMPORTANT
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.
- [x] Loadouts use standard [filename format](https://github.com/clustermod/HCMF3-Loadouts/wiki#standardization).
- [x] Loadouts are listed and documented in the respective readme file.

### CODE REVIEW

**Comment by PervonHarke** [comment]
This might close the linked issue regarding documentation. Though this PR will not completely resolve that specific issue, 
it may need reopening once this PR is merged.

Or it will be unlinked before merging to prevent that.

**Comment by PervonHarke** [comment]
Possible thing to do is to change all "unknown" entries for the loadout versions to 1.3 since that has been the latest version for  a long time and basically for every loadout in this folder.

**Review by Tapawingo — APPROVED** [review]

File: loadouts/germany/loadouts/deu_2012_ksk_tropentarn.sqf [hunk 1]
```diff
@@ -249,7 +249,7 @@ if (_condition && _role in ["FWP"]) then
 };
 
 // EOD  , HAS TO BE SET AS EOD INSIDE 3DEN EDITOR THROUGH ACE.
```
[the remark this answers is not in the document]
Comment (Tapawingo) [reply loadouts/germany/loadouts/deu_2012_ksk_tropentarn.sqf:251]: Similarly you can do the same for engineers:
```sqf
_unit setVariable ["ace_isEngineer", 1, true];
```
`0`: Untrained, only be able to replace wheels, based on settings.
`1`: Engineer.
`2`: Advanced Engineer, allowing a higher level of repairs

[the remark this answers is not in the document]
Comment (PervonHarke) [reply loadouts/germany/loadouts/deu_2012_ksk_tropentarn.sqf:251]: Will keep this in mind for future loadouts. Thank you.

**Pull request merged**